### PR TITLE
Search Runs cleaned up error messages

### PR DESCRIPTION
**Pull request by mparkhe**
- Fixing spacing in UI
- Fixing typos in error message

### CODE REVIEW

**Comment by mparkhe** [comment]
cc: @Zangr

**Comment by mparkhe** [comment]
image

image

File: mlflow/server/js/src/components/ExperimentView.css [hunk 1]
```diff
@@ -132,11 +132,12 @@
...
 
 div.error-message {
-  margin-left: 100px;
+  margin-top: 8px;
+  margin-left: 0;
   /*width: auto;*/
 }
 span.error-message {
-  color: red;
+  color: #DB1905;
```
Comment (andrewmchen) [inline mlflow/server/js/src/components/ExperimentView.css]: micronit: usually these are lowercased.

Comment (mparkhe) [reply mlflow/server/js/src/components/ExperimentView.css]: That's legit..

**Comment by gioa** [comment]
LGTM!

**Comment by Zangr** [comment]
LGTM

**Review by Zangr — APPROVED** [review]
LGTM

**Pull request merged**